src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -658,13 +659,17 @@ namespace AirLoopHVACDOAS {
 
                     } else if (SELECT_CASE_var == "AIRLOOPHVAC:UNITARYSYSTEM") {
                         OutsideAirSys(thisDOAS.m_OASystemNum).InletNodeNum(CompNum) =
-                            OutsideAirSys(thisDOAS.m_OASystemNum).compPointer[CompNum]->AirInNode;
+                            OutsideAirSys(thisDOAS.m_OASystemNum)
+                                .compPointer[CompNum]
+                                ->getAirInNode(OutsideAirSys(thisDOAS.m_OASystemNum).ComponentName(CompNum), 0);
```

Comment:
Switched from a UnitarySys pointer with direct access to UnitarySys member data to DataHVACSystem::HVACSystem data pointer with member function to access air inlet node.